ci/long_running_tests/check_workloads.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+ROOT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
+
+pushd "$ROOT_DIR"
+
+echo "WORKLOAD 1"
```

Comment:
In this script and the other one, you can use a bash while loop instead:
```
for workload in ci/workloads/*; do
    echo "Workload $workload"
    ....
done
```

So adding a workload is just dropping a new file in the folder.

Reply:
+1 yeah; also, it would provide more clarity than just having Workload1, Workload2, etc as identifiers